tests/Agent/IntegrationTests/IntegrationTestHelpers/ProfilerLogFile.cs

```diff
@@ -48,7 +48,7 @@ public ProfilerLogFile(string logDirectoryPath, TimeSpan? timeoutOrZero = null,
                 Thread.Sleep(TimeSpan.FromSeconds(1));
             } while (timeTaken.Elapsed < timeout);
```

Comment:
Does this need the same logic as in `AgentLogFile.cs`?

Reply:
No current test cases need it, but it makes sense to make it the same between the two classes. 👍